app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -78,6 +85,15 @@ def available_images
       client.images
     end
 
+    def create_disk(size, image_uuid)
+      client.disks.create(
+        :name         => "foreman-#{Time.now.to_i}",
+        :size_gb      => size.to_i,
+        :zone_name    => zone,
+        :source_image => client.images.select { |i| i.id == image_uuid }.first.name
```

Comment:
Not by ID, only by name, which doesn't really work in our case because the image model stores the UUID.